documentation/modules/con-securing-client-authentication.adoc

```diff
@@ -103,8 +103,10 @@ metadata:
 type: Opaque
 data:
   password: Z2VuZXJhdGVkcGFzc3dvcmQ= <1>
+  sasl.jaas.config: b3JnLmFwYWNoZS5rYWZrYS5jb21tb24uc2VjdXJpdHkuc2NyYW0uU2NyYW1Mb2dpbk1vZHVsZSByZXF1aXJlZCB1c2VybmFtZT0ibXktdXNlciIgcGFzc3dvcmQ9ImdlbmVyYXRlZHBhc3N3b3JkIjsK <2>
 ----
 <1> The generated password, base64 encoded.
+<2> The `sasl.jaas.config` string, base64 encoded.
```

Comment:
I would not use the field name to explain the field itself. It would be better saying something like "The JAAS configuration string for SASL SCRAM-SHA-512 authentication".